src/components/connect-create-buttons.tsx

```diff
@@ -1,71 +1,84 @@
 import { __ } from '@wordpress/i18n';
+import { useOffline } from '../hooks/use-offline';
 import { cx } from '../lib/cx';
 import { getIpcApi } from '../lib/get-ipc-api';
 import { ArrowIcon } from './arrow-icon';
 import Button, { ButtonVariant } from './button';
 import offlineIcon from './offline-icon';
 import { Tooltip } from './tooltip';
 
-interface ConnectCreateButtonsProps {
-	connectSite: () => void;
-	isOffline: boolean;
+interface ConnectButtonProps {
 	connectButtonVariant: ButtonVariant;
-	createButtonVariant: ButtonVariant;
+	connectSite?: () => void;
 	disableConnectButtonStyle?: boolean;
+	className?: string;
+}
+
+interface CreateButtonProps {
+	createButtonVariant: ButtonVariant;
 	selectedSite: SiteDetails;
+	createButtonText?: string;
+	className?: string;
```

Comment:
I think that since we now have two separate components, we could consider simplifying the props naming for both buttons, e.g.:

```suggestion
	variant: ButtonVariant;
	selectedSite: SiteDetails;
	text?: string;
	className?: string;
```

Reply:
Thank you for the suggestions! Makes sense. I will apply the improvements :)

Reply:
Updated in https://github.com/Automattic/studio/pull/740/commits/ab3a4a0ad8c17008bbac0ba0c31d756de2300479